app/(frontend)/new-poll/client.tsx

```diff
@@ -296,11 +297,13 @@ const NewPollPageClient = ({ canCreatePoll }: { canCreatePoll: boolean }) => {
   };
 
   // Update slug when title changes, if slug is empty
-
+  const slug = form.watch("slug");
   const onBlurTitle = () => {
-    if (form.formState.dirtyFields.title && !form.getValues("slug")) {
-      form.setValue("slug", slugify(form.getValues("title").toLowerCase()));
-      form.trigger("slug");
+    if (formState.dirtyFields.title && !slug) {
+      form.setValue("slug", slugify(form.getValues("title").toLowerCase()), {
+        shouldValidate: true,
+        shouldTouch: true,
+      });
     }
   };
```

Comment:
I actually think it's fine if this replaces the slug every-time the user blurs the title. In other words, no need to check `&& !slug` before setting the title.

Reply:
Alright